frontend/language/src/nb.json

```diff
@@ -951,6 +951,11 @@
   "schema_editor.maxLength": "Maksimal lengde",
   "schema_editor.maximum_exclusive": "Mindre enn ",
```

Comment:
Er det et mellomrom for mye før " her?

Reply:
Ja, men tror det er av en grunn. Her tror jeg det legges til et tall etter teksten, og derfor plassert et mellomrom i enden.